Johnsona Procedure 1 PR - Installing Wi-Fi Router

alyssajohnsonunofficial opened the pull request:
# Procedure 1 - Installing a Wi-Fi Router

I began my first procedure, which is for beginner-level users who want to install a Synology WRX560 Wi-Fi router in their home. 


**Completed Work**
- Started on my first procedure: 'installnewwifirouter'
- Procedure's goal: To successfully install a Synology WRX560 Wi-Fi router. 
- I laid out the basic sections with headers, added some text and images, and made empty numbered lists that my steps will go into. 


# User Scenario - First-Time Installer

**Scenario**: "Alyssa and Kaiya are college roommates in their early 20s. They need to set up a Wi-Fi router to get internet access for their school and work devices, but neither of them have ever installed one before and they don't have a lot of experience with installing hardware or with the terminology used to describe Wi-Fi routers. They just purchased a Synology WRX560 router and want quick, simple instructions."


## Notable Review Requests
- Should I pad out that introductory text under the title a little bit? I feel like it's super blunt and quick, but I didn't want to add too much filler or give the procedure a tone that doesn't fit the assignment. 
- How does the layout look so far? I hope to add the actual steps and more content soon when I have the time, but I tried to go ahead and situate the structure now. It was also important to me to have a section for terminology, since that fits the beginner-level audience of this procedure.
- Are the photos sufficient? This one's a bit tough to get creative photos for, so the only idea I have so far is just showing what the specific Wi-Fi router looks like from the front and back.

Eden-Miller commented:
Hi Lauren!

I hope you will find the following information helpful: 

- Your staging move is a lot more clear here! I would consider reformatting. I don't think it should be italicized, but that's a personal preference. It might be helpful to give this information in list format. 
- I like the way you broke up your sections using clear titles. 
- I can't really identify any alerting moves. I would add at least one to try to bypass some user pain points. If you do add one, make sure the formatting of all alerting moves are consistent so they are easily identifiable to the reader. 
- Your coaching moves are very effective! I love your use of images and labeling! The arrows and boxes help a lot, especially if the reader is more reading to do oriented. The color choice is also cute considering your used NCSU branding. I like the approach. For my procedure, I used a contrasting color to the NCSU red (since my procedure is focused about the NCSU TCA website), so you could see the boxes better, but your use of arrows helps mitigate that potential issue. 

It looks great Lauren!

Thank you! 
Eden Miller

alyssajohnsonunofficial committed:
Created files

alyssajohnsonunofficial committed:
Work and PR for Procedure 1

alyssajohnsonunofficial committed:
Small change

gmsimon22 commented on johnsona/installnewwifirouter.md:
```diff
@@ -0,0 +1,32 @@
+# How to Install a Synology WRX560 Wi-Fi Router
+
+This beginner-friendly procedure will guide you through the process of installing a Synology WRX560 Wi-Fi router. 
```
Per your review request question, I think adding a bit extra in the introductory description would be nice. It's simple and good as is, but maybe you could add any features of the router (wide-range capabilities or whatnot) that make it stand out as a good choice

WBlan97 replied:
I agree! Although I do think the motivation and staging doesn't have to be super long because the goal is fairly self-explanatory.

gmsimon22 commented on johnsona/installnewwifirouter.md:
```diff
@@ -0,0 +1,32 @@
+
+![Back side of a Synology WRX560](./../johnsona/assets/images/7-1080.edc11c9a.jpg)
+
+## Terms to Know
```
I love that you are adding a terminology section. I would definitely need this if I were setting up a wifi router

WBlan97 replied:
I second that!

dmbrasw2 replied:
I agree! This will make the guide more approachable by more levels of users

gmsimon22 commented on johnsona/installnewwifirouter.md:
```diff
@@ -0,0 +1,32 @@
+
+1. 
+2. 
+3. 
```
For the photos, if you refer to something specific in the back then maybe you can highlight or zoom in on it.

dmbrasw2 replied:
Screenshots of the setup software will be great for this part

gmsimon22 commented on johnsona/installnewwifirouter.md:
```diff
@@ -0,0 +1,32 @@
+
+1. 
+2. 
+3. 
```
I think the structure is good! Since it's beginner-friendly, the terms section helps to cover prerequisite knowledge that the user might be missing

WBlan97 replied:
I think the structure seems good!

gmsimon22 reviewed (commented):
Looking good so far! I left my comments addressing your questions.

WBlan97 commented on johnsona/installnewwifirouter.md:
```diff
@@ -0,0 +1,32 @@
+- An ISP Modem
+- A working outlet 
+- A device for set-up (phone, laptop, desktop, etc.)
+
```
Maybe you could even work some of these prerequisites into the staging section.

dmbrasw2 replied:
I agree!

WBlan97 commented:
Structure seems to be all the way there! Now just have to fill in the coaching stages.

alyssajohnsonunofficial committed:
Filled Out User Scenario Section

Copy and pasted the user scenario to its respective section so I could have it for reference

alyssajohnsonunofficial committed:
Fleshed Out Intro and Added Terms to Know

I fleshed out my introduction to be more in line with the staging phase, and then I added terms (not defined yet) to my "Terms to Know" section.

alyssajohnsonunofficial committed:
Filled Out My Terms Section

I defined my terms in "Terms to Know", moved a photo to the top of the page, and adjusted some language

alyssajohnsonunofficial committed:
Added a new screenshot and finished first half of hardware instructions

I filled out the first half of my hardware instructions and added a new screenshot

alyssajohnsonunofficial committed:
Finished the hardware instructions

alyssajohnsonunofficial committed:
Added another screenshot and filled out my software instructions and conclusion

I added a third screenshot, filled out my software set-up instructions, and included a wrap-up section

alyssajohnsonunofficial committed:
Added my user scenario

alyssajohnsonunofficial committed:
Moved image code and added screenshot code

I moved the codelines for my images to the images folder file and added the codelines of my screenshots to the screenshots folder file for my own reference.

lingeringcode committed:
Feedback from Lindgren
